src/benchmarks/notcoin/results_gas.json

```
@@ -0,0 +1,22 @@
+{
+  "results": [
+    {
+      "label": "FunC",
+      "gas": {
+        "transfer": "16984",
+        "burn": "12732",
+        "discovery": "6545"
+      },
+      "pr": "https://github.com/OpenBuilders/notcoin-contract"
+    },
+    {
+      "label": "1.6.2",
+      "pr": "https://github.com/tact-lang/tact/pull/2229",
```

```suggestion
      "pr": "https://github.com/tact-lang/tact/pull/2329",
```

The #2229 didn't feature notcoin

---

Thanks, fixed